src/iso_alloc.c

```diff
@@ -364,12 +356,12 @@ INTERNAL_HIDDEN void create_canary_chunks(iso_alloc_zone_t *zone) {
     /* This function is only ever called during zone
      * initialization so we don't need to check the
      * current state of any chunks, they're all free.
-     * It's possible the call to rand_uint64() here will
+     * It's possible the call to rand() here will
```

Comment:
There are no calls to `rand()` here.

Reply:
Its a called to `us_rand_uint64` but it should be corrected.